react/DeliveryItems.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { FC } from 'react'
+
+import ProductList from './components/ProductList'
+import ShippingHeader from './components/DeliveryHeader'
+import { useOrder } from './components/OrderContext'
+
+const Shipping: FC = () => {
+  const { deliveryParcels, giftRegistryData } = useOrder()
+
+  if (deliveryParcels.length === 0) {
+    return null
```

Comment:
Isn't it better to show something to user indicating that this doesn't have parcels? (Just questioning)

Reply:
Nope! As it currently happens, we shouldn't show a parcel type section if the order doesn't have any parcels of that specific type (delivery, pickup and takeaway)